app/src/androidTest/java/fiofoundation/io/fiokotlinsdktestapp/DevSdkTests.kt

```diff
@@ -11,6 +11,7 @@ import fiofoundation.io.fiosdk.errors.FIOError
 import fiofoundation.io.fiosdk.errors.session.TransactionBroadCastError
```

Comment:
Eric to review

Reply:
Reviewed the tests. We do not want to go overboard with tests, but it would be good to have one positive and one negative test. I would suggest:
1. Modify this test to include multiple lock periods.
2. Create a test that catches trying to add a lock to an existing account.

Reply:
I will extend these tests to do those...